rasa/core/training/interactive.py

```diff
@@ -785,7 +788,17 @@ def _collect_actions(events: List[Dict[Text, Any]]) -> List[Dict[Text, Any]]:
     with open(export_story_path, append_write, encoding="utf-8") as f:
         for conversation in sub_conversations:
             parsed_events = rasa.core.events.deserialise_events(conversation)
-            s = Story.from_events(parsed_events)
+            applied_events = []
+
+            for i, event in enumerate(parsed_events):
```

Comment:
hm is there a way we could make this code https://github.com/RasaHQ/rasa/blob/master/rasa/core/trackers.py#L335 through 362 reusable, instead of reinventing the wheel here? Maybe it should go in events instead of under trackers, e.g. a method that takes a list of events and returns applied ones, this can then be used in the tracker method. i'd prefer that method because it seems to handle UURs better -- the [:-2] has failed me before (I still have a PR floating somewhere because buttons in CLI don't show up after some UURs or restarts for this reason)